src/overlays/actors/ovl_En_Twig/z_en_twig.c

```diff
@@ -5,17 +5,30 @@
  */
 
 #include "z_en_twig.h"
+#include "assets/objects/object_twig/object_twig.h"
 
 #define FLAGS (ACTOR_FLAG_10)
 
 #define THIS ((EnTwig*)thisx)
 
+#define GET_PARAM1(ring) ((ring)->dyna.actor.params & 0xF)
+#define GET_PARAM2(ring) (((ring)->dyna.actor.params >> 4) & 0x1F)
+#define GET_PARAM3(ring) (((ring)->dyna.actor.params >> 9) & 0x7F)
```

Comment:
```suggestion
#define RING_GET_PARAM1(thisx) ((thisx)->params & 0xF)
#define RING_GET_PARAM2(thisx) (((thisx)->params >> 4) & 0x1F)
#define RING_GET_PARAM3(thisx) (((thisx)->params >> 9) & 0x7F)
```
You'll also want to adjust this so it takes `thisx`. I wonder if the prefix `RING` is too generic as well